src/vaishnaviVJun20/Bank.java

```diff
@@ -0,0 +1,64 @@
+package vaishnaviVJun20;
+
+public class Bank {   
+	int creditCount = 0;
+	int debitCount = 0;
+	int printBalanceCount = 0;
+	static int creditCount1 = 0;
+	static int debitCount1 = 0;
+	static int printBalanceCount1 = 0;
```

Comment:
Remove the initialization of all instance variables.

Reply:
Done